launcher/modManager/cmodmanager.cpp

```diff
@@ -112,7 +112,7 @@ void CModManager::loadMods()
 			auto json = JsonUtils::toJson(mod);
 			json["localSizeBytes"].Float() = total;
 			if(!name.is_absolute())
-				json["storedLocaly"].Bool() = true;
+				json["storedLocally"].Bool() = true;
```

Comment:
Would this cause issues with existing JSON files?

Reply:
Safe. Unused code actually. Property is never read.